### ISSUE TITLE
[BUG] osrelease not upgraded during "bastille upgrade TARGET"

### ISSUE THREAD

**Issue by michael-o** [BastilleBSD/bastille#668]
When [searching for `osrelease`](https://github.com/search?q=repo%3ABastilleBSD%2Fbastille%20osrelease&type=code) in the code it os only touched during creation of a jail. When then jail is updated:
* thick: it is not touched
* thin: release is updated or re-mounted, it is not touched

I believe that at the current state it will rather provide invalid information after some time.

**Comment by bmac2** [comment]
@michael-o is this still a valid issue?  If so, how do i recreate it?

@yaazkal @tschettervictor

**Comment by michael-o** [comment]
> @michael-o is this still a valid issue? If so, how do i recreate it?
> 
> @yaazkal @tschettervictor

Yes, here is the spot: https://github.com/BastilleBSD/bastille/blob/1dedfed3a01e73697b21ffb9c12c769cecc28bfa/usr/local/share/bastille/create.sh#L188. There is not other spot when it is modified.

Instruction:
* Create a thick jail with 13.3-RELEASE
* Check `jail.conf` for `osrelease`
* Upgrade jail to 13.4-RELEASE
* Check `jail.conf` for `osrelease` it will still say 13.3-RELEASE

**Comment by michael-o** [comment]
Note that due to #658 the upgrade command is unusable for me.

**Comment by michael-o** [comment]
https://github.com/BastilleBSD/bastille/pull/811 addresses this issue.
